gomuko/__tests__/gameLogic.test.ts

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable jest/no-commented-out-tests */
+import {deepEquals} from '../../common/common';
+import {Board, createNewBoard} from '../gameLogic';
+
+describe('Game Logic Test', () => {
+  it('Create Board', function () {
+    const board: Board = createNewBoard(9);
+    const expectedBoard: Board = [
+      ['', '', '', '', '', '', '', '', ''],
+      ['', '', '', '', '', '', '', '', ''],
+      ['', '', '', '', '', '', '', '', ''],
+      ['', '', '', '', '', '', '', '', ''],
+      ['', '', '', '', '', '', '', '', ''],
+      ['', '', '', '', '', '', '', '', ''],
+      ['', '', '', '', '', '', '', '', ''],
+      ['', '', '', '', '', '', '', '', ''],
+      ['', '', '', '', '', '', '', '', ''],
+    ];
+
+    expect(deepEquals(board, expectedBoard)).toBe(true);
+  });
+
+  // it('Initial move', function () {
```

Comment:
add a LOT more unit tests...

Reply:
understandable its is as we noted in class the last thing we need to check while fixing the ai again. 
will do once the ai is fixed and all is well.